scripts/fvt-tools.sh

```diff
@@ -62,11 +62,10 @@ exec_in_hostnet_of_node() {
 
 get_default_gw() {
   local nodename=${1}
+  v="default"
   if [[ "${PROVIDER}" == "ibmcloud" ]]; then
     provider_type=$(get_provider_type)
-    [[ ${provider_type} == "softlayer" ]] && v="10.0.0.0/8" || v="default"
-  else
-    v="default"
+    [[ ${provider_type} == "softlayer" ]] && v="10.0.0.0/8"
```

Comment:
We can combine two expressions into one line like `if [[ "${PROVIDER}" == "ibmcloud" ]] && [[ "$(get_provider_type)" == "softlayer" ]]; then v="10.0.0.0/8"; fi`